ENIGMAsystem/SHELL/Platforms/SDL/Event.cpp

```diff
@@ -253,7 +253,6 @@ void SDL_Event_Handler::mouseButtonDown(const SDL_Event *event) {
 void SDL_Event_Handler::mouseButtonUp(const SDL_Event *event) {
   int btn = SDL_map_button_enum(event->button.button);
   if (btn < 0) return;
-  SDL_CaptureMouse(SDL_FALSE);
```

Comment:
Apparently, we do not need to explicitly tell SDL to end mouse capture. It already tracks in its Windows video backend the count of mouse buttons that are pressed, and automatically revokes the mouse capture when all the mouse buttons are released. Simply having this line is what caused SDL to have the anomaly to begin with.

Reply:
It would be particularly shitty if I have to come back and change this for xlib later or something because SDL does not provide a wParam equivalent for its mouse events, despite using those same modifiers internally. You would have to request the entire mouse state.